source/extensions/filters/http/custom_response/policies/local_response_policy.cc

```diff
@@ -0,0 +1,88 @@
+#include "source/extensions/filters/http/custom_response/policies/local_response_policy.h"
+
+#include "envoy/stream_info/filter_state.h"
+
+#include "source/common/common/enum_to_int.h"
+#include "source/common/config/datasource.h"
+#include "source/common/formatter/substitution_format_string.h"
+#include "source/common/formatter/substitution_formatter.h"
+#include "source/common/http/header_map_impl.h"
+#include "source/common/http/message_impl.h"
+#include "source/common/http/utility.h"
+#include "source/common/router/header_parser.h"
+#include "source/extensions/filters/http/custom_response/custom_response_filter.h"
+
```

Comment:
Some of the code below looks similar to [`BodyFormatter`](https://github.com/envoyproxy/envoy/blob/7805e46b0a5a27bd390c151b6ba03862331a39cf/source/common/local_reply/local_reply.cc#L19) and `LocalReplyImpl`. Should/could the code be refactored to ensure consistent behavior?